src/envoy/http/jwt_auth/pubkey_cache.h

```diff
@@ -46,6 +48,18 @@ class PubkeyCacheItem {
     for (const auto& aud : jwt_config_.audiences()) {
       audiences_.insert(SanitizeAudience(aud));
     }
+
+    if (jwt_config_.has_local_jwks() &&
```

Comment:
Instead of the whole block here, just use [`Envoy::Config::Datastore::read`](https://github.com/envoyproxy/envoy/blob/master/source/common/config/datasource.h#L16) and you're good with all case.

Reply:
Done